src/ImageSharp/Colors/Spaces/YCbCr.cs

```diff
@@ -117,13 +121,7 @@ public YCbCr(byte y, byte cb, byte cr)
         /// <inheritdoc/>
         public override int GetHashCode()
         {
-            unchecked
-            {
-                int hashCode = this.Y.GetHashCode();
-                hashCode = (hashCode * 397) ^ this.Cb.GetHashCode();
-                hashCode = (hashCode * 397) ^ this.Cr.GetHashCode();
-                return hashCode;
-            }
+            return GetHashCode(this);
```

Comment:
I completely agree, I was just following the pattern the other color spaces use. I can fix all of them while I'm at it, I thought maybe there was a reason for it 😄